ansible/hosts.sandbox

```diff
@@ -1,5 +1,5 @@
 [simple]
-ec2-13-234-35-248.ap-south-1.compute.amazonaws.com
+ec2-13-127-221-249.ap-south-1.compute.amazonaws.com
```

Comment:
I guess these changes will unblock deployments to sandbox and QA?

Reply:
:+1: